src/base/dev/vga/vgaemu.c

```diff
@@ -2525,6 +2526,7 @@ int vgaemu_map_bank()
 #endif
 
   i = vga_emu_map(VGAEMU_MAP_BANK_MODE, first);
+  invalidate_unprotected_page_cache(0xa0000, 0x20000);
```

Comment:
Maybe we need e_invalidate_full() here,
and hide the page cache stuff entirely
into the e_invalidate macros?

Reply:
Sure I can do that for now, since the read_* and write_* functions are only called by simx86. 

Eventually they can also be useful for internal read/write purposes outside cpuemu (e.g. in instremu.c because they are safer than READ_BYTE and friends), but that needs some other changes anyway so I'll leave that for later.

Reply:
Done